homeassistant/components/enphase_envoy/strings.json

```diff
@@ -337,6 +337,30 @@
       },
       "configured_reserve_soc": {
         "name": "Configured reserve battery level"
+      },
+      "acb_battery_state": {
+        "name": "Battery state",
+        "state": {
+          "discharging": "discharging",
+          "idle": "idle",
+          "charging": "charging",
+          "full": "full"
```

Comment:
Let's capitalize the first letter

Also, idle is a common key. I don't have the exact one on me but I can send you that one tomorrow if you didn't find it yet